python/GafferSceneTest/ShaderAssignmentTest.py

```diff
@@ -215,8 +215,8 @@ def testInputAcceptanceFromBoxes( self ) :
 		# this because the backwards compatibility is provided only when
 		# a script is loading.
 
-		s.execute( """script["b"]["a"]["shader"].setInput( script["b"]["in"] )""" )
-		s.execute( """script["a"]["shader"].setInput( script["b"]["out"] )""" )
+		s["b"]["a"]["shader"].setInput( s["b"]["in"] )
+		s["a"]["shader"].setInput( s["b"]["out"] )
```

Comment:
The comment above this sections needs updating now `execute` has been removed

Reply:
Thanks for spotting that - fixed in c5c816f.